hid-kye.c

```diff
@@ -483,6 +483,91 @@ static __u8 pensketch_m912_rdesc_fixed[] = {
+	0x95, 0x04,         /*          Report Count (4),           */
+	0x81, 0x03,         /*          Input (Constant, Variable), */
+	0x09, 0x32,         /*          Usage (In Range),           */
+	0x14,               /*          Logical Minimum (0),        */
+	0x25, 0x01,         /*          Logical Maximum (1),        */
+	0x95, 0x01,         /*          Report Count (1),           */
+	0x81, 0x02,         /*          Input (Variable),           */
+	0x05, 0x01,         /*          Usage Page (Desktop),       */
+	0x09, 0x30,         /*          Usage (X),                  */
+	0x75, 0x10,         /*          Report Size (16),           */
+	0x95, 0x01,         /*          Report Count (1),           */
+	0xA4,               /*          Push,                       */
+	0x55, 0x0D,         /*          Unit Exponent (13),         */
+	0x65, 0x33,         /*          Unit (Inch^3),              */
+	0x14,               /*          Logical Minimum (0),        */
+	0x26, 0x00, 0x3C,   /*          Logical Maximum (15360),    */
+	0x34,               /*          Physical Minimum (0),       */
+	0x46, 0x00, 0x3C,   /*          Physical Maximum (15360),   */
+	0x81, 0x02,         /*          Input (Variable),           */
+	0x09, 0x31,         /*          Usage (Y),                  */
+	0x26, 0x00, 0x28,   /*          Logical Maximum (10240),    */
+	0x46, 0x00, 0x28,   /*          Physical Maximum (10240),   */
+	0x81, 0x02,         /*          Input (Variable),           */
+	0x05, 0x0D,         /*          Usage Page (Digitizer),     */
+	0x09, 0x30,         /*          Usage (Tip Pressure),       */
+	0x14,               /*          Logical Minimum (0),        */
+	0x26, 0xFF, 0x03,   /*          Logical Maximum (1023),     */
+	0x34,               /*          Physical Minimum (0),       */
+	0x46, 0xFF, 0x03,   /*          Physical Maximum (1023),    */
+	0x81, 0x02,         /*          Input (Variable),           */
+	0x75, 0x10,         /*          Report Size (16),           */
+	0xC0,               /*      End Collection,                 */
+	0xC0,               /*  End Collection                      */
+	0x05, 0x0C,         /*  Usage Page (Consumer),              */
+	0x09, 0x01,         /*  Usage (Consumer Control),           */
+	0xA1, 0x01,         /*  Collection (Application),           */
+	0x85, 0x12,         /*      Report ID (18),                 */
+	0x14,               /*      Logical Minimum (0),            */
+	0x25, 0x01,         /*      Logical Maximum (1),            */
+	0x75, 0x01,         /*      Report Size (1),                */
+	0x95, 0x04,         /*      Report Count (4),               */
+	0x0A, 0x79, 0x02,   /*      Usage (AC Redo Or Repeat),      */
+	0x0A, 0x1A, 0x02,   /*      Usage (AC Undo),                */
+	0x0A, 0x2D, 0x02,   /*      Usage (AC Zoom In),             */
+	0x0A, 0x2E, 0x02,   /*      Usage (AC Zoom Out),            */
+	0x81, 0x02,         /*      Input (Variable),               */
```

Comment:
You don't need to change Report Size past this point. Just leave it at 1, as set above, and specify the appropriate report count to ignore the rest of the report.